### ISSUE TITLE
RSC-005 error on refining the dc:rights metadata

### ISSUE THREAD

**Issue by chector77** [w3c/epubcheck#1242]
In epubcheck 4.2.5
This metadata 
```xml
<dc:rights xmlns:html="http://www.w3.org/1999/xhtml" id="x1234">Copyright © 2022</dc:rights>
<meta xmlns:html="http://www.w3.org/1999/xhtml" refines="#x1234" property="role">copyrightDate</meta>
```

yields this error:
```
ERROR(RSC-005): c:/books/cv_pxe_master/books/70010LTI17418/notneeded/dev/etext2/9780136690337_et2_dccsb_cu_redserif_default.epub/OPS/package.opf(34,121): Error while parsing file: Property "role" must refine a "creator" or "contributor" property. 
```

This is a new error and the metadata has previously worked.

(@rdeltour: edited markup for code blocks)

**Comment by mattgarrish** [comment]
Why not avoid refines entirely and just add the date using the similarly-named dcterms property?

```
<meta property="dcterms:dateCopyrighted">2022</meta>
```

Using the `refines` attribute typically verges on useless for reading systems, as they rarely do anything with linked properties.

But, in any case, the `role` property is for describing entities (people and organizations), not concepts like dates. That restriction's as old as the property itself: http://idpf.org/epub/30/spec/epub30-publications.html#role

You could always suppress the error if this is for internal use, but as far as I see epubcheck is reporting the issue correctly. You'd need to open an issue in the [epub-specs repository](https://github.com/w3c/epub-specs/issues) if you want the restrictions reconsidered.

**Comment by rdeltour** [comment]
Nothing to add to what @mattgarrish said 👍

I'm closing this issue as `invalid` since it is not a bug with EPUBCheck. But feel free to keep commenting here!
